files/.prettierignore

```diff
@@ -1,19 +1,10 @@
 # Prettier is also run from each package, so the ignores here
 # protect against files that may not be within a package
 
-# dependencies
-/node_modules/
-
 # misc
 !.*
-.eslintcache
 .lint-todo/
 
 # ember-try
 /.node_modules.ember-try/
-/bower.json.ember-try
-/npm-shrinkwrap.json.ember-try
-/package.json.ember-try
-/package-lock.json.ember-try
-/yarn.lock.ember-try
-/pnpm-lock.yaml.ember-try
+/pnpm-lock.ember-try.yaml
```

Comment:
The lock file for pnpm _does_ end with `.yaml` for readability reasons:
https://github.com/ember-cli/ember-try/blob/master/lib/dependency-manager-adapters/pnpm.js#L15C31-L15C55